src/components/Table/Column.tsx

```diff
@@ -41,10 +40,12 @@ export const Cell = forwardRef<HTMLTableCellElement, ColumnProps>(({ children, .
 
 const ColumnElement = forwardRef<HTMLTableDataCellElement | HTMLTableHeaderCellElement, ColumnElementProps>(
   ({ alignment, element, className, children }: ColumnProps & ColumnElementProps, ref) => {
+    const alignmentClass = alignment === 'left' ? styles.Left : alignment === 'center' ? styles.Center : styles.Right;
```

Comment:
I would generally advise targeting this through attributes, e.g.

```scss
.Table {
    *[data-alignment="left"] {}
    *[data-alignment="right"] {}
}
```

But this is all good for now!